### PR TITLE
Car Port for Hyundai Santa FE Hybrid 2023 (non-HDA2)

### PR DESCRIPTION
**Pull request by wileytom65**

### CODE REVIEW

**Comment by sshane** [comment]
What is your dongle id?

**Comment by wileytom65** [comment]
I believe it's:
826b9eb4ac54a1c2
-Tom

Sent from Yahoo Mail on Android 
 
  On Wed, May 24, 2023 at 12:49, Shane ***@***.***> wrote:   


What is your dongle id?

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -845,25 +845,62 @@ class Buttons:
...
     (Ecu.fwdRadar, 0x7d0, None): [
       b'\xf1\x00TMhe SCC FHCUP      1.00 1.00 99110-CL500         ',
     ],
+    (Ecu.fwdRadar, 0x7e1, None): [
+      b'\xf1\x00TMhe SCC FHCUP      1.00 1.00 99110-CL500         ', 
+      b'"\xf1\x00',
+    ],
+    (Ecu.fwdRadar, 0x7e1, None): [
+      b'\xf1\x8799110CL500\xf1\x00TMhe SCC FHCUP      1.00 1.00 99110-CL500         ', 
+      b'"\xf1\x87\xf1\x81\xf1\x00',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:860]: Not sure where any of these address/FW version pairs came from, going to revert and re-do for you

**Comment by sshane** [comment]
Fixed the FW versions. FYI, you're on dashcam, so you'll need to switch to master or master-ci to use stock OP!

**Comment by wileytom65** [comment]
They came from the fingerprinting, under carParams for this vehicle.  I added them per the instructions and have been driving with them for some time.
-Tom

    On Wednesday, May 24, 2023 at 04:12:45 PM AKDT, Shane Smiskol ***@***.***> wrote:  
 
 


@sshane commented on this pull request.

In selfdrive/car/hyundai/values.py:
> +    (Ecu.fwdRadar, 0x7e1, None): [
+      b'\xf1\x00TMhe SCC FHCUP      1.00 1.00 99110-CL500         ', 
+      b'"\xf1\x00',
+    ],
+    (Ecu.fwdRadar, 0x7e1, None): [
+      b'\xf1\x8799110CL500\xf1\x00TMhe SCC FHCUP      1.00 1.00 99110-CL500         ', 
+      b'"\xf1\x87\xf1\x81\xf1\x00',

Not sure where any of these address/FW version pairs came from, going to revert and re-do for you

**Comment by sshane** [comment]
@wileytom65 Do you remember which harness you used?

**Comment by wileytom65** [comment]
The 'L harness'
-Tom

    On Tuesday, January 2, 2024 at 09:20:50 PM AKST, Shane Smiskol ***@***.***> wrote:  
 
 


@wileytom65 Do you remember which harness you used?

**Pull request merged**